src/components/metrics.tsx

```diff
@@ -436,6 +439,30 @@ export const QueryTable: React.FC<QueryTableProps> = ({ index, namespace }) => {
     }
   }
 
+  const getCsvColumns = () => {
+    const csvColumns = [];
+    for (let i = 1; i < columns.length; i++) {
+      const column = columns[i];
+      if (typeof column?.title === 'string') {
+        csvColumns.push(column.title);
+      } else if (column.title?.props?.children) {
+        csvColumns.push(column.title.props.children);
+      }
+    }
+    return csvColumns;
+  };
+
+  const getCsvRows = () => {
+    const csvRows = [];
+    for (let i = 0; i < rows.length; i++) {
+      const row = [...rows[i]].slice(1);
+      csvRows.push(row);
+    }
+    return csvRows;
+  };
```

Comment:
I guess this would be a little nicer if we use `map` instead of a `for` loop.

Also, I think `[...rows[i]].slice(1)` is equivalent to just `rows[i].slice(1)` since both `...` and `slice` create [shallow copies](https://developer.mozilla.org/en-US/docs/Glossary/Shallow_copy) of the array.

So we'd end up with just `const getCsvRows = () => rows.map((row) => row.slice(1));`

Reply:
Ah, I see! Thank you for the correction. I updated this in the latest commit.